scripts/plugins/deploy_plugins.py

```diff
@@ -222,6 +223,11 @@ def deploy_plugins():  # pylint: disable=too-many-locals,too-many-branches,too-m
             logger.debug("running bitops schema parsing...")
             cli_config_list, _ = get_config_list(opsrepo_config_file, plugin_schema_file)
 
+            # Compose a CLI and export it as "BITOPS_{PLUGIN_NAME}_CLI}"
+            cli = PluginConfigCLI(cli_config_list).get_command()
+            os.environ[f"BITOPS_{plugin_name.upper()}_CLI"] = cli
+            logger.debug(f"Exported CLI: {cli}")
```

Comment:
Keeping in line with *almost all* other log event in BitOps I would suggest you change `{}` to `[]`